portal-api/src/v1/gef/models/facilities.js

```diff
@@ -220,15 +220,15 @@ class Facility {
       throw new InvalidParameterError('isUsingFacilityEndDate', req.isUsingFacilityEndDate);
     }
 
-    if ('facilityEndDate' in req && 'bankReviewDate' in req) {
+    if (req.facilityEndDate && req.bankReviewDate) {
```

Comment:
I think the reason we're using `in` here is to check the existence of the property :) has this been changed because were not deleting the field in the database?

Reply:
Reverting these changes as I'm no longer setting both equal to null